esp/templates/program/modules/studentregtwophase/rank_classes.html

```diff
@@ -73,5 +73,9 @@
     <i class="star-fill icon-2x icon-star" data-bind="visible: interested"></i>
   </a>
   <label>Click the star to mark this class as interested.</label>
+  <br />
+  {% for pri, pri_display in priorities %}
+  <button data-bind="click: function(data) { $parent.prioritySelection[{{pri}}-1](data.id); $j('#pri-{{pri}}').trigger('change.select2'); }">Set as {{pri_display}}</button>
```

Comment:
I don't know knockout super well, but is it possible to not have this be a giant inline function?  Even just defining it somewhere else and referencing here seems nicer to me, if that's not too hard.